src/timeman.cpp

```diff
@@ -28,6 +28,8 @@
 
 namespace Stockfish {
 
+const static double plyExtra[] = {0.95, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0};
```

Comment:
This doesn't seem to be needed for the current PR

Reply:
Sure. I think we could remove all the originalPly stuff as well, but have pushed a version just removing this array.